### PR TITLE
Refactor TOC for management libraries

### PR DESCRIPTION
**Pull request by nickzhums**
changes are
1 use consistent name "resource management" for all management libraries
2 remove redundant tocs 
3 add track 2 sdk readmes to all available RPs that are GA-ed

### CODE REVIEW

File: docs-ref-mapping/reference-unified.yml [hunk 1]
```diff
@@ -267,26 +251,24 @@
...
-      landingPageType: Service
-      children:
-      - com.microsoft.azure.management.containerregistry*
     - name: Resource Management
       uid: azure.java.sdk.landingpage.services.containerregistry.ResourceManagement
       landingPageType: Service
       children:
-      - com.azure.resourcemanager.containerregistry*   
+      - com.azure.resourcemanager.containerregistry*  
+      - com.microsoft.azure.management.containerregistry*
```
Comment (yungezz) [inline docs-ref-mapping/reference-unified.yml:259]: is this still necessary?

Comment (nickzhums) [reply docs-ref-mapping/reference-unified.yml:259]: this service is not GA-ed yet so we put both Track 1 and Track 2 here still

**Comment by scbedd** [comment]
@sima-zhu take a look at this?

File: docs-ref-mapping/reference-unified.yml [hunk 2]
```diff
@@ -626,7 +598,7 @@
     uid: azure.java.sdk.landingpage.services.management
     landingPageType: Service
     items:
-    - name: Management
+    - name: Resource Management
       uid: azure.java.sdk.landingpage.services.management.Management
```
Comment (sima-zhu) [inline docs-ref-mapping/reference-unified.yml:602]: No need to change uid to ResourceManagement?

File: docs-ref-mapping/reference-unified.yml [hunk 3]
```diff
@@ -635,7 +607,7 @@
     uid: azure.java.sdk.landingpage.services.mediaservices
     landingPageType: Service
     items:
-    - name: Management
+    - name: Resource Management
       uid: azure.java.sdk.landingpage.services.management.mediaservices
```
Comment (sima-zhu) [inline docs-ref-mapping/reference-unified.yml:611]: No need to change uid to ResourceManagement?

**Review by sima-zhu — APPROVED** [review]
Overall looks good to me.

**Comment by sima-zhu** [comment]
The PR wants to merge to master. Could you change to smoke-test? We merge changes to smoke-test first, and we are responsible to merge smoke-test to master. We use the gating branch to prevent potential doc issues.

**Comment by nickzhums** [comment]
> The PR wants to merge to master. Could you change to smoke-test? We merge changes to smoke-test first, and we are responsible to merge smoke-test to master. We use the gating branch to prevent potential doc issues.

done

**Pull request merged**